### PR TITLE
Add back spec that got lost somehow

### PR DESCRIPTION
**Pull request by white238**
... i fail.

### CODE REVIEW

**Comment by white238** [comment]
LGTM

File: scripts/llnl_scripts/llnl_lc_build_tools.py [hunk 1]
```diff
@@ -271,6 +271,7 @@ def uberenv_create_mirror(prefix, spec, project_file, mirror_path):
     assertUberenvExists()
     cmd  = "python {0} --create-mirror -k ".format(get_uberenv_path())
     cmd += " --prefix=\"{0}\" --mirror=\"{1}\"".format(prefix, mirror_path)
+    cmd += "--spec=\"{0}\" ".format(spec)
```
Comment (kennyweiss) [inline scripts/llnl_scripts/llnl_lc_build_tools.py:274]: Is this missing a space? 
```suggestion
    cmd += " --spec=\"{0}\" ".format(spec)
```

**Review by kennyweiss — APPROVED** [review]

File: scripts/llnl_scripts/llnl_lc_build_tools.py (hunk 1 above)
Comment (white238) [reply scripts/llnl_scripts/llnl_lc_build_tools.py:274]: Fixed the line above it to be more consistent with all other lines.. good catch though.

**Comment by white238** [comment]
LGTM

**Comment by white238** [comment]
LGTM

**Review by agcapps — APPROVED** [review]
All I can say, Chris, is thanks for whacking those moles.

**Pull request merged**